parent/pom.xml

```diff
@@ -1295,6 +1295,40 @@
       </build>
     </profile>
 
+    <profile>
+      <id>skip-random-tests</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/*RandomizedPropertyTest.java</exclude>
+              </excludes>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>include-random-tests</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <includes>
+                <include>**/*RandomizedPropertyTest.java</include>
+              </includes>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
```

Comment:
I don't really understand why we need this extra profile. AFAIK it should be picked-up for the default run, and can be excluded by activating the `skip-random-tests` profile. Is there something I missed?

Reply:
I think the name is maybe suboptimal. A better name would be "run random tests exclusively". By default the include pattern is something like "**/*Test.java" which would include those tests. That's why we have the exclude profile to not run those tests. 

Now, this profile is in order to only run these tests. This config is effectively overwriting the default pattern, and with this profile only this specific pattern is matched against.

At least this is how I think it works. I still don't fully understand how configurations are merged when you specify several profiles, like we do.

Reply:
>Now, this profile is in order to only run these tests.

Ah! Maybe add a comment above the profile describing what you can use this for. That would explain for me why it makes sense to have a separate profile

> I still don't fully understand how configurations are merged when you specify several profiles

You're not alone. Generally, with excludes I assume they take some form of precedence, but I really don't know what happens with combinations of conflicting includes and excludes. But you've already convinced me on the additional profile ;)